styleguide/examples/links/example_links.html

```diff
@@ -0,0 +1,13 @@
+<div class="cads-prose">
```

Comment:
This got auto-formatted when running `./bin/generate-examples`. Left it in this commit as it's small.